cloud/aws/templates/aws_oidc/variable_definitions.json

```diff
@@ -31,24 +31,6 @@
     "tfvar": true,
     "type": "string"
   },
-  "APPLICANT_OIDC_CLIENT_ID": {
-    "required": true,
-    "secret": true,
-    "tfvar": true,
-    "type": "string"
-  },
-  "APPLICANT_OIDC_CLIENT_SECRET": {
-    "required": true,
-    "secret": true,
-    "tfvar": true,
-    "type": "string"
-  },
-  "APPLICANT_OIDC_DISCOVERY_URI": {
```

Comment:
discovery url shouldn't be a secret. we should remove it from secrets and keep it here.

Reply:
Done. I'll remove it from secrets and switch to this in follow up PR to keep this one no-op.